pom.xml

```diff
@@ -212,6 +212,7 @@
 										<Implementation-Title>${project.name}</Implementation-Title>
 										<Implementation-Version>${project.version}</Implementation-Version>
 										<Implementation-Vendor>${project.organization.name}</Implementation-Vendor>
+										<Add-Opens>java.base/java.lang java.base/java.util</Add-Opens>
```

Comment:
reference: https://openjdk.org/jeps/261#Breaking-encapsulation:~:text=Packaging%3A%20Modular%20JAR%20files

Reply:
--add-exports add exports is open access to public field and methods. but it not open access to non-public fields/methods. --add-opens will open non-public fields/methods for reflection usage.

We need --add-opens since we need access to non-public methods such - setAccessible(true)
https://github.com/Azure/azure-functions-java-worker/blob/72ae44af798fa8beed2bc43e6295cd6cdfc65032/src/main/java/com/microsoft/azure/functions/worker/handler/FunctionEnvironmentReloadRequestHandler.java#L54